### PR TITLE
Add I18N tutorial FAQ link

### PR DESCRIPTION
**Pull request by HansMuller**
Added a link to the I18N tutorial to the FAQ.

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -467,7 +467,7 @@ to the mobile platform using a
 [`BasicMessageChannel`](https://docs.flutter.io/flutter/services/BasicMessageChannel-class.html).
 
 Learn more about accessing platform and third-party services in Flutter with
-[platform channels](/platform-channels/).
+[platform channels](https://flutter.io/platform-channels/).
```
Comment (sethladd) [inline faq.md]: let's remove the https://flutter.io part. That makes it hard to stage an in-progress version of the site.

File: faq.md [hunk 2]
```diff
@@ -482,16 +482,14 @@ what you might need reflection/mirrors for – please let us know at
 
 ### How do I do internationalization (i18n), localization (l10n), and accessibility (a11y) in Flutter?
 
-Flutter has basic support for accessibility on iOS and Android, though this
-feature is a work in progress.
+Learn more about i18n and l10n in the
+[Flutter Internationalization Tutorial](https://flutter.io/tutorials/internationalization/).
```
Comment (sethladd) [inline faq.md]: Please remove https://flutter.io from the URL.

As it is, it makes it hard to stage the in-progress site.

File: faq.md (hunk 1 above)
Comment (HansMuller) [reply faq.md]: Sorry, for some reason the URL checked kept failing on the the relative URL (until it didn't). I'll fix the URLs

**Comment by sethladd** [comment]
lgtm!

**Pull request merged**